src/arch/svm/utils.ts

```diff
@@ -19,6 +19,7 @@ import { SvmSpokeClient } from "@across-protocol/contracts";
 import { FillType, RelayData } from "../../interfaces";
 import { BigNumber, SvmAddress, getRelayDataHash, isUint8Array } from "../../utils";
 import { EventName, SVMEventNames, SVMProvider } from "./types";
+import { intToU8Array32 } from "@across-protocol/contracts/dist/src/svm/web3-v1";
```

Comment:
Not sure we should use helper methods from contracts if they were not exported from the package root. And we need to handle `bigint` here anyway so maybe its ok to rewrite it here.